### PR TITLE
Ride buttons

### PR DESCRIPTION
**Pull request by avivbarniv**
## ℹ️ Change description:
Nothing here

<details><summary><sub>templates<sub></summary>
<p>
  
<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

</p>
</details>

### CODE REVIEW

File: examples/client/Locomotion/src/Components/CardRow/index.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+              {true || (paymentMethod && moment(paymentMethod.expiresAt).isBefore(moment())) ? <Error>{i18n.t('payments.expired')}</Error> : null}
+              {true || (paymentMethod && paymentMethod.hasOutstandingBalance) ? <Error>{i18n.t('payments.hasOutstandingBalance')}</Error> : null}
+            </>
+          )}
+      </TextContainer>
+    </Container>
+  </TouchableOpacity>
+);
+
+// {"brand": "visa", "createdAt": "2022-06-20T07:34:04.506Z", "customerId": "596950b9-4199-470d-9968-ac9db79a37d3", "deletedAt": null, "expiresAt": "2034-12-31T23:59:59.999Z", "hasOutstandingBalance": false, "id": "97f00c47-7eeb-4854-bdbc-aeec26ad449f", "isDefault": false, "isExpired": false, "lastFour": "4242", "name": "visa", "stripeId": "pm_1LCf7XIaHOVjSfrG0KbEBC6r", "updatedAt": "2022-06-20T07:34:04.506Z"}
```
Comment (lieblichaaron) [inline examples/client/Locomotion/src/Components/CardRow/index.tsx]: this here for a reason?

File: examples/client/Locomotion/src/pages/ActiveRide/newMap.js [hunk 2]
```diff
@@ -56,12 +56,12 @@ export default React.forwardRef(({
     }
   };
 
-  const buildAvailabilityVehicles = () => availabilityVehicles.map(vehicle => (
+  const buildAvailabilityVehicles = () => (currentBsPage === 'main' ? availabilityVehicles.map(vehicle => (
```
Comment (dorshay6) [inline examples/client/Locomotion/src/pages/ActiveRide/newMap.js]: wanna take out to a varible?

**Review by dorshay6 — APPROVED** [review]

File: examples/client/Locomotion/src/context/newRideContext/index.js [hunk 3]
```diff
@@ -30,9 +30,48 @@ function useInterval(callback, delay) {
...
+});
 const HISTORY_RECORDS_NUM = 10;
 
+export const latLngToAddress = async (lat, lng) => {
+  const location = `${lat},${lng}`;
+  const data = await getGeocode({
+    latlng: location,
+  });
+
+  console.log('data', data);
```
Comment (lieblichaaron) [inline examples/client/Locomotion/src/context/newRideContext/index.js]: remove

File: examples/client/Locomotion/src/pages/ActiveRide/newMap.js [hunk 4]
```diff
@@ -115,20 +115,24 @@ export default React.forwardRef(({
...
         followsUserLocation={isUserLocationFocused}
         moveOnMarkerPress={false}
-        onRegionChange={(event) => {
+        onRegionChangeComplete={async (event) => {
           if (selectLocationMode) {
             const { latitude, longitude } = event;
+            const lat = latitude.toFixed(6);
+            const lng = longitude.toFixed(6);
+            const description = await latLngToAddress(lat, lng);
             saveSelectedLocation({
```
Comment (lieblichaaron) [inline examples/client/Locomotion/src/pages/ActiveRide/newMap.js:133]: add streetAddress field

**Review by lieblichaaron — APPROVED** [review]

**Pull request merged**